pkg/oplabels/labels.go

```diff
@@ -17,16 +18,17 @@ const (
 	NameLabelValue             = "cassandra"
 	InstanceLabel              = "app.kubernetes.io/instance"
 	VersionLabel               = "app.kubernetes.io/version"
-	CreatedByLabel             = "app.kubernetes.io/created-by"
+	emptyString                = ""
+	labelCut                   = ".-_"
 )
 
+var blacklistRegex = regexp.MustCompile(`[^a-zA-Z\d_.-]`)
```

Comment:
See notes below, but instead of blacklisting, make this regexp the one that accepts everything - and if possible, use the Kubernetes' internal regexp that's used for validation.